testcases/open_posix_testsuite/conformance/interfaces/clock_nanosleep/2-2.c

```diff
@@ -95,8 +96,9 @@ int main(void)
 			printf("Test PASSED\n");
 			return PTS_PASS;
 		} else {
-			printf("Slept for too long: %d >= %d\n",
-			       (int)tsafter.tv_sec, sleepuntilsec);
+			printf("Slept for too long: %lld >= %ld\n",
+			       (long long int)tsafter.tv_sec,
+				   (long long int)sleepuntilsec);
```

Comment:
Shouldn't the string be "%lld >= %lld\n", since you are casting both to long long?

Reply:
You are right, thank I'll fix it